packages/styles/stories/Components/skeleton/circle.example.html

```diff
@@ -0,0 +1,17 @@
+<div class="fd-skeleton fd-skeleton--animated" style="width: 50px; height: 50px;">
+    <svg width="100%" height="100%" class="fd-skeleton__canvas">
+        <defs>
+            <linearGradient id="skeletonGradient">
+                <stop offset="0" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
+                <stop class="fd-skeleton__middle">
+                    <animate attributeName="offset" values="0; 0; 0.5; 1; 1" dur="2s" repeatCount="indefinite"></animate>
+                </stop>
+                <stop offset="1" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
```

Comment:
for me this is very weird. Why do you set the `stop-color` for the first and last stop in the markup and for the middle you use a class?

Reply:
added a class for it